.taprc

```diff
@@ -24,8 +24,19 @@ files:
   - ./packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-validator-besu/v21-get-block-endpoint.test.ts
   - ./packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-validator-besu/get-past-logs-endpoint.test.ts
   - ./packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-validator-besu/get-transaction-endpoint.test.ts
+<<<<<<< HEAD
```

Comment:
@krriscode I just noticed these, please do a conflict resolution here so that the diff only includes your changes.

Reply:
@petermetz, I have resolved the conflict of .taprc file, please re-review it.

Reply:
@krriscode Thank you! Looks good!